aiaccel/apps/optimize.py

```diff
@@ -73,23 +74,47 @@ def main() -> None:
     parser.add_argument("--config", nargs="?", default=None)
     parser.add_argument("--executor", nargs="?", default="local")
     parser.add_argument("--resume", action="store_true", default=False)
-
+    parser.add_argument(
+        "--fix", nargs="*", default=[], help="Parameter names to fix with best values from previous study"
+    )
     args, unk_args = parser.parse_known_args()
-    config = oc.merge(oc.load(args.config), oc.from_cli(unk_args))
 
-    if "storage" not in config.study:
-        config.study.storage = {
+    config = oc.merge(oc.load(args.config), oc.from_cli(unk_args))
+    config.study.setdefault(
+        "storage",
+        {
             "_target_": "optuna.storages.RDBStorage",
             "url": "sqlite:///optuna.db",
             "engine_kwargs": {"connect_args": {"timeout": 30}},
-        }
-
-    if "study_name" not in config.study:
-        config.study.study_name = "aiaccel_study"
+        },
+    )
+    config.study.setdefault("study_name", "aiaccel_study")
```

Comment:
こちらの方がロジック見えやすくて良いかと思いますが，いかがですか？
```suggestion
    if "study_name" not in config.study:
        config.study.study_name = "aiaccel_study"
```

Reply:
その記述へ変更後、 complexity が上がって `ruff check` でエラーが出るようになってしまいましたが、ひとまず可読性を優先してその記述へ変更し、 `noqa: C901` で`optimize.py` の `main()` はこのチェックを無視するようにしています。

将来的に、ruff での complexity に関するチェックは除外してもいいかもです。